src/app/schedule-view/component.ts

```diff
@@ -92,7 +92,12 @@ export class ScheduleViewComponent implements OnInit, OnDestroy {
           } as ScheduleEvent;
           scheduleEvents.push(event);
         }
-        ++color;
+        if (color == 6){
```

Comment:
Once the colors array is a class member in the Schedule, you can change this to check against the length of the colors array instead of hardcoding `6`

Reply:
Cant seem to find the write code. Blanking on what I need to call to get the correct value. I am calling Schedule.colors.length, but I get an error saying `'colors' does not exist in 'typeof Schedule'`